os/kernel/binary_manager/binary_manager_binfile.c

```diff
@@ -92,10 +93,11 @@ static int binary_manager_clear_binfile(int bin_idx)
  *	 This function scans user binary files.
  *
  ****************************************************************************/
-void binary_manager_scan_ubin(void)
+void binary_manager_scan_ubinfs(void)
```

Comment:
How about changing the function name as `binary_manager_update_ubin_table`?
It can be confused because we have similar function, `binary_manager_scan_ubin`. It is not easy to distinguish.